fastapi/app/api/api_v1/endpoints/colormap.py

```diff
@@ -26,3 +26,7 @@ async def generate_colormap_response(plot_config: Dict, bp_file, variable: str):
     }
 
     return MsgpackResponse(content=mesh_json)
+
+
+async def generate_colormap_data(plot_config: Dict, bp_file, variable: str):
+    return
```

Comment:
Whats going on here?

Reply:
Empty function as a placeholder for generating the images for the colormap/heatmap plots. I created it as I worked out the flow of everything but we don't need it yet (obviously). Removed for now.